src/PlatformEngines.jl

```diff
@@ -347,13 +434,14 @@ function probe_platform_engines!(;verbose::Bool = false)
     end
 
     # Search for a compression engine
-    for (test, unpack, package, list, parse) in compression_engines
+    for (test, unpack, package, list, parse, parse_symlinks) in compression_engines
         if probe_cmd(`$test`; verbose=verbose)
             # Set our compression command generators
             gen_unpack_cmd = unpack
```

Comment:
Since you're changing the signature of `gen_unpack_cmd` here by adding an `excludelist` option, you should add that to the generic version up top that just errors out, so that we don't get a MethodError in the future.